src/models/Offer/Model.ts

```diff
@@ -150,18 +150,30 @@ export class Offer extends Model<Offer> {
   public getApprovalEvents: HasManyGetAssociationsMixin<OfferApprovalEvent>;
 
   public expireForStudents = () => {
-    this.studentsExpirationDateTime = moment().startOf("day").toDate();
+    this.studentsExpirationDateTime = moment().startOf("day").subtract(1, "day").toDate();
   };
 
   public expireForGraduates = () => {
-    this.graduatesExpirationDateTime = moment().startOf("day").toDate();
+    this.graduatesExpirationDateTime = moment().startOf("day").subtract(1, "day").toDate();
   };
 
+  public expire() {
+    if (this.targetApplicantType === ApplicantType.graduate) this.expireForGraduates();
+    if (this.targetApplicantType === ApplicantType.student) this.expireForStudents();
+    if (this.targetApplicantType === ApplicantType.both) {
+      this.expireForGraduates();
+      this.expireForStudents();
+    }
+    return this;
```

Comment:
Es necesario retornar this?

Reply:
No, pero me pareció lo mas comodo que devuelva el objeto con todo expirado

Reply:
pero comodo para que? osea por lo que no estamos usando lo que retorna expire